src/lib/holocene/api-pagination.svelte

```diff
@@ -237,7 +237,7 @@
       />
     {/if}
     <nav
-      class="flex shrink-0 flex-col gap-4 md:flex-row"
+      class="flex shrink-0 flex-col gap-4 sm:flex-row"
```

Comment:
This seems to work a bit better with responsiveness. Give it a try and let me know what you think! 
```suggestion
      class="flex gap-4 flex-row flex-wrap justify-center"
```